#!/usr/bin/env python
PACKAGE = "aseia_car_sim"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("time_sigma"       , double_t, 0, "Sigma of Time normal distributed error"       ,  0   ,  0,  10   )
gen.add("distance_sigma"   , double_t, 0, "Sigma of Distance normal distributed error"   ,  0   ,  0,  50   )
gen.add("position_sigma"   , double_t, 0, "Sigma of Position normal distributed error"   ,  0   ,  0, 100   )
gen.add("orientation_sigma", double_t, 0, "Sigma of Orientation normal distributed error",  0   ,  0,   3.14)
gen.add("confidence"       , double_t, 0, "Confidence to transform sigmas to intervals"   , 0.99,  0,   1   )

exit(gen.generate(PACKAGE, "aseia_car_sim", "Error"))
